src/enums/regions.ts

```diff
@@ -88,5 +94,8 @@ export default [
   europeCentral2,          
   europeWest2,            
   europeWest3,            
-  europeWest6,    
+  europeWest6,  
+  // multi-region
+  eu,
```

Comment:
Hmm can we just auto include them somehow? I dont know if ppl will know to scroll down and include these for big query

Reply:
maybe based on what regions they select. i.e. if they select any us based region we include us and if they include any eu based region we include eu

Reply:
I think people are going to be confused if they select big query as a resource to scan but arent getting any back because they arent familiar with the multi region concept and how that maps to cloudgraph. Also, we would be querying "us" or "eu" as regions for all the resources even though those concepts only apply to BQ (as far as I know). So the current behavior is that if they select big query, we get all the big query resources in that project regardless of what regions they live in correct? Maybe that is the correct way to handle this and its not an actual bug 🤔 . My main concern with this solution is it would be confusing to users as to what "US" and "EU" are if they are not familiar with the multi-region concept of BQ.